pandas/io/tests/parser/common.py

```diff
@@ -836,7 +836,8 @@ def test_integer_overflow_bug(self):
         result = self.read_csv(StringIO(data), header=None, sep=' ')
         self.assertTrue(result[0].dtype == np.float64)
 
-        result = self.read_csv(StringIO(data), header=None, sep='\s+')
+        result = self.read_csv(StringIO(data), header=None,
```

Comment:
My change causes an error to be raised whenever the `sep` parameter is regex and `quoting != csv.QUOTE_NONE` for the Python engine.  This test will break as a result for the <u>Python engine</u> without specifying that `quoting=csv.QUOTE_NONE`.